contributors/guide/pull-requests.md

```diff
@@ -590,7 +597,33 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
-## Fixing linter issues
+## Large or Automatic Edits
+
+Some tools make it very easy to create large Pull Requests, for example:
+- global search/replace
+- linters which automatically correct issues (see also next section)
+- large language models (LLMs) which generate code or documentation
+
+To make it easier for reviewers to handle such Pull Requests, please explain
+how it was generated in the "Special notes for your reviewer" section of the
+Pull Request description. Reviewers may then be able to reproduce those steps
+(search/replace, linters) or can start the review with the right expectations
+(LLMs). Also consider the section about [splitting up Pull
+Requests](#dont-open-pull-requests-that-span-the-whole-repository) above.
+
+Even with such tools it is still your responsibility as submitter of a Pull
+Request to ensure that the change is correct (to the best of your knowledge),
+and that making the change improves the project enough to justify the cost that
+is needed to review and merge the Pull Request (see previous section). If
+unsure, add a "RFC: " (request for comments) or "WIP: " (work in progress) prefix
```

Comment:
WIP: has meaning to our CI in a way RFC: does not

Reply:
They are both not mentioned elsewhere in the document. I was wondering whether they deserve a more complete description and then decided against it because this didn't look like the right place for it. But perhaps a few more sentence are fine?

> RFC is merely informational. WIP also prevents merging accidentally. To avoid running tests before the Pull Request is ready, create a Draft Pull Request, in which case `/test` can be used to trigger only specific test jobs.

Reply:
Agreed, mentioning one option here that has the desired effect is better than trying to cover all of them. Changed to:

> If unsure, create a draft Pull Request and ask for guidance.

Reply:
I was unsure whether I should use "Draft Pull Request" or "draft Pull Request". The document uses "Pull Request", but it wasn't entirely clear to me why capital letter where chosen - I just did the same to be consistent. But "draft" is not used elsewhere, so I decided to use the normal spelling.

Reply:
Let's have a debate over a single character difference :stuck_out_tongue_closed_eyes: ...

Reply:
That's styling in the web UI. In normal prose, GitHub doesn't capitalize: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews

But I agree, we probably used "Pull Request" because of the button, so let's stay consistent. Changed...